readme.html

```diff
@@ -5,25 +5,18 @@
 -->
 
-<h2>Getting ERS credentials</h2>
-<ol>
-    <li>
-        <h3>Enable Ers</h3>
-        <p>ERS uses on HTTPS port 9060 which is by default closed. Clients trying to access this port without enabling ERS first, will face a timeout from the server.
-            Therefore, the first requirement is to enable ERS from the Cisco ISE admin UI. Go to <b>Administration > Settings > ERS Settings</b> and enable the Enable ERS
-            for Read/Write radio button</p>
-    </li>
-    <li>
-        <h3>Creating ERS Admin</h3>
-        <p>Go to <b>Administration > Settings > ERS Settings</b> and then from the panel on the left select <b>Admin Users</b> under administrators.
-            Now add an account by clicking <b>Add > Create an admin user</b>. Then enter name and password and select <b>ERS Admin</b> in Admin Group and then press save.</p>
-    </li>
-</ol>
+<h2>Asset Configuration For Authentication</h2>
+<ul>
+    <li>ERS uses HTTPS port 9060 which is by default closed. Clients trying to access this port without enabling ERS first will face a timeout from the server. Therefore, the first requirement is to enable ERS from the Cisco ISE admin UI. Go to Administration > Settings > ERS Settings and enable the Enable ERS for Read/Write radio button</li>
+    <li>Go to Administration > System > Admin Users. Now add an account by clicking Add > Create an admin user. Then enter name and password and select ERS Admin in Admin Group and then press save.</li>
+    <li>Go to Administration > System > Admin Users. Now add an account by clicking Add > Create an admin user. Then enter name and password and select MnT Admin in Admin Group and then press save.</li>
+    <li>Configurations expect user with MnT Admin Access group in username/password fields and user in ERS Admin group in ERS username/password fields or user with both MnT Admin or ERS Admin access group in username/password field.</li>
+    <li>Also, you can add both MnT Admin and ERS Admin Access groups to a user and use that credentials in username/password. The App will use username/password if ERS username/password is not provided</li>
+</ul>
 <h2>Note</h2>
 <ol>
-    <li>Quarantine device and Unquarantine device actions may not work properly sometimes. Apply policy and Clear policy with policy type QUARANTINE are recommended to use</li>
-    <li>ERS credentials are required for actions list endpoints, get device info, update device info, get resources, delete resource, create resource, update resource,
-        apply policy and create policy</li>
+    <li>Actions quarantine system and unquarantine system are removed in the version X.X.X. Users are advised to use apply policy and clear policy actions to achieve the same objective</li>
+    <li>ERS credentials are required for actions list endpoints, get device info, update device info, get resources, delete resource, create resource, update resource, apply policy and create policy</li>
```

Comment:
this list is long enough that it may be worth creating a sublist of the actions instead of having them on one line for readability?